### PR TITLE
chore: add config/readme

### PR DESCRIPTION
**Pull request by notrab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| Commerce   | [View schema](/commerce/grafbase/schema.graphql)   | [![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Blog&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Ftemplates%2Ftree%2Fmain%2Fcommerce)   |
+| HackerNews | [View schema](/hackernews/grafbase/schema.graphql) | [![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Blog&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Ftemplates%2Ftree%2Fmain%2Fhackernews) |
+| Todo       | [View schema](/todo/grafbase/schema.graphql)       | [![Deploy to Grafbase](https://grafbase.com/button)](https://grafbase.com/new/configure?template=Blog&source=https%3A%2F%2Fgithub.com%2Fgrafbase%2Ftemplates%2Ftree%2Fmain%2Ftodo)       |
+
+## Learn more
+
+- <a href="https://grafbase.com/docs"><strong>Documentation</strong></a>
+- <strong>[Templates](templates)</strong>
+- <strong>[CLI](cli)</strong>
+- <strong>[Examples](examples)</strong>
```
Comment (fbjork) [inline README.md]: link to the new repo here.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+root = true
```
Comment (fbjork) [inline .editorconfig:1]: ignore this file no?

File: README.md (hunk 1 above)
Comment (notrab) [reply README.md]: Already done

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<p align="center">
+  <a href="https://grafbase.com">
+    <img src="https://grafbase.com/images/other/grafbase-logo-circle.png" height="96">
+    <h3 align="center">Grafbase</h3>
+  </a>
+</p>
+
+<p align="center">
+  The data platform for developers.
```
Comment (fbjork) [inline README.md:9]:
```suggestion
 GraphQL backends made easy.
```

File: .editorconfig (hunk 2 above)
Comment (notrab) [reply .editorconfig:1]: We can (it's only inserting new lines) tbh

**Review by vitalikda — APPROVED** [review]

**Pull request merged**